main.py

```diff
@@ -281,6 +283,7 @@ def command_skip():
 def play_next_song(e: BaseException = None, skip_count: int = 0):
     async def inner_f():
         global current_channel_content
+        global current_bust_content
```

Comment:
We could derive this from `current_channel_content` as well, without adding another global variable?

```python
for _, local_filepath, _ in current_channel_content:
    os.remove(local_filepath)
```

Reply:
current_channel_content is treated like a queue, where the next song is popped off the front. Its empty state signals the end of the bust. I wasn't sure if this was preferred so I left that pattern untouched.

Reply:
Aha! We do pop from it, I had forgotten that. Definitely not something to rely upon then!